### DISCUSSION TITLE
is object spawn removed permanently? any way to enable

### DISCUSSION THREAD

**Discussion by ravi3517** [YimMenu/YimMenu#3506] [Q&A]
any way to get back object spawn tab.

**Comment by lonelybud** [comment]
the object spawner tab you are mentioning https://github.com/YimMenu/YimMenu/discussions/1918 never merged in the menu. It was just a showcase and is no longer maintained. I think someone might have tried to recreate something like it in lua -

here (I have not worked on it)
https://github.com/Deadlineem/Extras-Addon-for-YimMenu/blob/main/Extras-Addon.lua

**Comment by snowleopardd** [comment]
If you only want an object spawner lua just use this, it's much better and less bloated
https://github.com/xesdoog/object-spawner

the extras addon one is more about money and general stuff, but gets the job done too
